### ISSUE TITLE
cannot run in php7.2 uper

### ISSUE THREAD

**Issue by fazelit** [crocodic-studio/crudbooster#1465]
please remove extention mcrypt

**Comment by geoffsari** [comment]
Yes, I faced a similar issue and downgraded to PHP 7.1.30 to get it working.

**Comment by alokraj68** [comment]
is this issue still there? I tried installing in 7.2 and got into an unknown issue. it's not installing crud booster via composer.
